tests/kani/AsyncAwait/spawn.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// compile-flags: --edition 2018
+// kani-flags: --enable-unstable --mir-linker
```

Comment:
btw, you don't need these flags any more. The MIR Linker is now the default one.

Reply:
A lot of tests still seem to have it. But I'll remove it.

Reply:
We haven't had time to remove them yet, but I think we should avoid adding them. We will probably fix the old tests once we remove the `--mir-linker` option which today is a no-op.

Reply:
A lot of tests still seem to have it. But I'll remove it.